docs/zh/operation/dataset_auto_scaling.md

```diff
@@ -0,0 +1,427 @@
+# 示例 - Dataset自定义弹性扩缩容
```

Comment:
Oops, missed the header in the last review.

Reply:
The problems have been modified,  thanks for your suggestions.